lib/Mojo/WebSocketProxy/Backend/JobAsync.pm

```diff
@@ -66,7 +66,7 @@ sub new {
     # We'd like to provide some flexibility for people trying to integrate this into
     # other systems, so any combination of Job::Async::Client, Job::Async and/or IO::Async::Loop
     # instance can be provided here.
-    $self->{client} //= do {
+    unless $self->{client} {
```

Comment:
Syntax error here, needs
```suggestion
    unless ($self->{client}) {
```

Reply:
IMHO this part doesnt need changing and should be kept as it is.